docs/roadmap/cosmos-hub-roadmap-2.0.md

```diff
@@ -13,7 +13,7 @@ The upgrades aim to add features such as liquidity, economic security, usability
   - Drives liquidity for tokens on the Cosmos Hub
   - Delivers price consistency and order execution
 
-## Vega Upgrade (expected Q4 2021)
+## Vega Upgrade (Completed Q4 2021)
```

Comment:
other completed milestones have dates, if we know the exact date, we can add it here